README.md

```diff
@@ -13,9 +13,38 @@ The telemetry client is a web frontend that displays data pushed from the server
 We use [yarn](https://yarnpkg.com/en/) to manage dependencies&mdash;it is compatible with `npm`, but is a little faster and allows us to specify a `yarn.lock` file.
 
 ```bash
+cd client/
+nvm use .
 yarn install
 ```
 
+### yarn/npm scripts
+All the below scripts can be run using ``npm`` or ``yarn``.
+
+#### Development
+```bash
+yarn run dev
+```
+
+#### Linting
+```bash
+yarn run lint
+```
+
+If you want to run a specific lint,
+
+```bash
+yarn run lint:js
+yarn run lint:ts
+yarn run lint:scss
+```
+
+#### Build
+
+```bash
+yarn run build
+```
+
```

Comment:
Want to add a section about starting the prod version with oneof php, python or go? They all have some kind of simple default http server.

Also are there any special rules for adding other .ts files? Would I just import them to app.ts?